ProjectSidewalk/SidewalkWebpage
# #2752 add amsterdam logos

Resolves #2752 

Adds logos for City of Amsterdam, Amsterdam Intelligence, and World Enabled to the Collaborators section of the home page for Amsterdam only. Note I changed the heights of the City of Amsterdam and Amsterdam Intelligence logos using CSS as they were very large otherwise.

##### Before/After screenshots (if applicable)
Before:
<img width="1440" alt="Screen Shot 2022-01-04 at 1 19 21 PM" src="https://user-images.githubusercontent.com/50668250/148111373-6f19f9f5-1254-47d8-91e3-b57ebb637b99.png">

After (on Amsterdam site):
<img width="1440" alt="Screen Shot 2022-01-05 at 2 09 26 PM" src="https://user-images.githubusercontent.com/50668250/148314881-59a1ca9f-4c21-480c-a09c-6c17c9f6d835.png">

After (on Columbus site):
<img width="1440" alt="Screen Shot 2022-01-05 at 2 12 05 PM" src="https://user-images.githubusercontent.com/50668250/148314903-641f2c55-3183-45da-9f91-8716505615ba.png">

##### Testing instructions
1. Modify the `SIDEWALK_CITY_ID` line in the `docker-compose.yml` to be `amsterdam`.
2. Run locally and navigate to the Collaborators section of the home page to see the added logos.
3. Modify the `SIDEWALK_CITY_ID` line in the `docker-compose.yml` to be an other city ID.
4. Run locally and navigate to the Collaborators section of the home page to see no added logos.
